### DISCUSSION TITLE
I can't run the source

### DISCUSSION THREAD

**Discussion by antonoius** [elsa-workflows/elsa-core#4099] [Q&A]
could you please help me to run the source code after cloning it, after clone and build and try to run dashboard project I got unreachable site error "This site can’t be reached"

**Comment by sfmskywalker** [comment]
Sure, Can you show me the exact steps that you did? Normally, all you need to do is:

1. Clone the repo.
2. Build the client assets (for the designer) using `build-assets.ps1` found at the root of the repo.
3. Run the dashboard project.
